frontend/src/components/InputBox.jsx

```diff
@@ -1,11 +1,11 @@
 const InputBox = ({ label, placeholder, name, value, handleFormChange }) => {
   return (
     <>
-      <label className="font-semibold ml-2">{`${label}`}</label>
+      <label className="font-semibold ml-2">{label}</label>
       <textarea
```

Comment:
could we have an option for the input box to render <input> instead of <textarea> so it supports password type?